src/grid/rtransform.py

```diff
@@ -52,7 +52,17 @@ def deriv2(self, x):
     def deriv3(self, x):
         """Abstract method for 3rd derivative of transformation."""
 
-    def transform_1d_grid(self, oned_grid):
+    @property
+    def tf_domain(self):
+        """tuple: Transformation domain."""
+        return self._tf_domain
+
+    @property
+    def tf_codomain(self):
+        """tuple: Transformation codomain."""
+        return self._tf_codomain
```

Comment:
Isn't it simpler ans easier to remember to just call them `domain` and `codomain`?